tests/unit/test__http.py

```diff
@@ -81,3 +81,115 @@ def test_build_api_url_w_extra_query_params(self):
         self.assertEqual(path, "/".join(["", "storage", conn.API_VERSION, "foo"]))
         parms = dict(parse_qsl(qs))
         self.assertEqual(parms["bar"], "baz")
+
+    def test_api_request_no_retry(self):
+        import requests
+
+        http = mock.create_autospec(requests.Session, instance=True)
+        client = mock.Mock(_http=http, spec=["_http"])
+
+        conn = self._make_one(client)
+        response = requests.Response()
+        response.status_code = 200
+        data = b"brent-spiner"
+        response._content = data
+        http.request.return_value = response
```

Comment:
this test is missing the comment the other tests have. There may be a way to refactor this also, but unsure if it is warranted.

Reply:
Which comment is missing?